app/domain/errors.rb

```diff
@@ -519,6 +519,27 @@ module AuthnJwt
         msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains claim name with spaces.",
         code: "CONJ00109E"
       )
+
+      MappedClaimsMissingInput = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claims value is empty, or was not found.",
+        code: "CONJ00108E"
+      )
+
+      MappedClaimsInvalidFormat = ::Util::TrackableErrorClass.new(
+        msg: "Mapped claims value '{0-mapped-claims-value}' is in invalid format.",
+        code: "CONJ00109E"
+      )
+
+      MappedClaimNotValidTuple = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claim value '{0-mapped-claim-value}'"\
+             " should have exactly one colon ':' separator",
+        code: "CONJ00110E"
+      )
+
+      MappedClaimDuplicationError = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claim '{0-purpose}' value '{1-claim-value}' appears twice or more",
```

Comment:
and here:
Failed to parse mapped claims, the claim '{0-purpose}' value '{1-claim-value}' appears twice or more